Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabLoader.cpp

```diff
@@ -277,6 +280,55 @@ namespace AzToolsFramework
             return !nestedTemplateReference->get().IsLoadedWithErrors();
         }
 
+        bool PrefabLoader::SanitizeLoadedTemplate(PrefabDomReference loadedTemplateDom)
+        {
+            if (!loadedTemplateDom)
+            {
+                return false;
+            }
+
+            Instance loadedPrefabInstance;
+            if (!PrefabDomUtils::LoadInstanceFromPrefabDom(loadedPrefabInstance, loadedTemplateDom->get()))
+            {
+                return false;
+            }
+
+            PrefabDom storedPrefabDom(&loadedTemplateDom->get().GetAllocator());
```

Comment:
nit: storedTemplateDom to be a bit pedantic.

Reply:
Variable was removed as per Ronald's suggestion.